docker-compose.yml

```
@@ -0,0 +1,82 @@
+services:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)